bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/component.jsx

```diff
@@ -79,8 +80,13 @@ class VideoListItem extends Component {
       actions,
       user,
       enableVideoStats,
+      usersLength,
     } = this.props;
 
+    if (usersLength < 3) {
+      actions.pop();
```

Comment:
I don't think you should use `pop()` in this way. Although it simulates the functionality your going for now (which I assume is removing the 'focus' action from the dropdown menu). If more actions are added to the menu from the `video-list` component, this will break.